doc/lua_api.md

```diff
@@ -490,6 +490,11 @@ to let the client generate textures on-the-fly.
 The modifiers are applied directly in sRGB colorspace,
 i.e. without gamma-correction.
 
+### Notes
+
+ * `TEXMOD_UPSCALE`: The texture with the lower resolution will be automatically
+   upscaled to the higher resolution texture.
```

Comment:
In general I'd say that documentation is not code and making it human-readable is more important than keeping it "DRY", but this style of documentation seems fine here.

What about `[png`? It uses `blitBaseImage` which calls `upscaleImagesToMatchLargest`.

Reply:
Good catch. Documented that one too. I agree that the documentation should not be cryptic, but if I'd inline this part, the documentation would become even harder to maintain.